packages/lib/src/interactions/Pan.tsx

```diff
@@ -8,13 +8,22 @@ import {
   useMoveCameraTo,
   useInteraction,
 } from './hooks';
+import { MouseButton } from './models';
 import type { CanvasEvent, Interaction } from './models';
 
-function Pan(props: Interaction) {
-  const { modifierKey } = props;
+interface Props extends Omit<Interaction, 'button'> {
+  button?: MouseButton;
```

Comment:
Since you've removed `Right` from `MouseButton`, the `Omit` is not needed here. Either re-add `Right` to the enum and change the prop here to `button?: MouseButton.Left | MouseButton.Right`, or leave `props: Interaction` as is.